cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -149,17 +149,8 @@ features:
         labelWithCount: "System Compute Profiles ({count})"
         import: Import
     Component-Overview:
-      emptyMessage: Available only in Distributed mode
       headers:
-        cdap: CDAP
-        cdh: CDH
-        hbase: HBASE
-        hdfs: HDFS
-        kafka: Kafka
-        spark: Spark
-        yarn: YARN
-        zookeeper: ZooKeeper
-      label: Component Overview
+        cdap: System Metrics
```

Comment:
This might cause issues with Administration when working with a cluster. In a cluster we will have platform details that will include `yarn`, `hbase`, `hdfs` etc., Is this change tested against a cluster?

Reply:
Also if we are removing `Component Overview` should this section be changed to something like `SystemServices` or something similar?

Reply:
Confirmed with @bdmogal, the backend PR for this JIRA (https://issues.cask.co/browse/CDAP-13360) will make it so that calls to `/system/serviceproviders` will always only return `cdap` in the response object. We're discontinuing support for the others in 5.0.

Have changed the i18n section label.